src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -1276,7 +1276,7 @@ public void TestConflictResolverExceptionsWrongIDThrown()
                 return new MutableDocument("wrong_id");
             });
             
-            TestConflictResolverExceptionThrown(wrongDocIDResolver, false);
+            //TestConflictResolverExceptionThrown(wrongDocIDResolver, false);
```

Comment:
This test should be modified to ensure the correct behavior.

Reply:
Yes, test name is wrong too. So this test will be removed. Additional tests will be added into another PR.